### DISCUSSION TITLE
Brightness not working

### DISCUSSION THREAD

**Discussion by ezrabutler** [tailwindlabs/tailwindcss#9479] [Help]
I'm trying to use two different brightness levels (`brightness-125` and `brightness-50`) to provide shading and depth to an SVG I'm creating.

On iMac Chrome - (both development and production) it works well.
On iMac Safari - (both development and production) it does not work at all.
On iPhone Chrome + Safari (production) it does not work at all.

The compiled/exported CSS includes
```
.brightness-125 {
  --tw-brightness: brightness(1.25);
  filter: var(--tw-blur) var(--tw-brightness) var(--tw-contrast)
    var(--tw-grayscale) var(--tw-hue-rotate) var(--tw-invert) var(--tw-saturate)
    var(--tw-sepia) var(--tw-drop-shadow);
}
.brightness-50 {
  --tw-brightness: brightness(0.5);
  filter: var(--tw-blur) var(--tw-brightness) var(--tw-contrast)
    var(--tw-grayscale) var(--tw-hue-rotate) var(--tw-invert) var(--tw-saturate)
    var(--tw-sepia) var(--tw-drop-shadow);
}
```
I've updated all of the various packages in my `package.json`
```
"autoprefixer": "^10.4.12",
    "postcss": "^8.4.17",
    "tailwindcss": "^3.1.8",
    "vite": "^3.0.7",
```

Thank you

**Comment by wongjn** [comment]
According to [MDN compatibility data](https://developer.mozilla.org/en-US/docs/Web/CSS/filter#browser_compatibility), for SVG elements, the `filter` CSS property should only work in Firefox so it is a little puzzling if it works in Chrome! In any case, since you are using it on SVGs, you could try using an [SVG filter](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/filter) instead.

**Comment by ezrabutler** [comment]
Thanks for pointing that out! It was a basic image, I just recreated it in
CSS/Tailwind, and it worked perfectly

Ezra Butler
347-277-2786
www.ezrabutler.com
